account_statement_import_camt/views/account_bank_statement_import.xml

```diff
@@ -11,6 +11,11 @@
                 <li>CAMT</li>
                 <li>zipped CAMT</li>
             </ul>
+            <ul id="statement_format" position="after">
+                <p
+                    id="checking_name_unique"
+                >Unique bank statement name checking (needed by CAMT importing) can be turned on at Settings > Invoicing > Bank Statement Import > Bank Statement Import.</p>
```

Comment:
This still sounds ambiguous to me. It sounds like it would work for all bank statement formats, and is required for CAMT statements, whereas effectively it only works for CAMT statements and it is not actually required for CAMT statements to be imported.